src/components/smart/OeActorWidgetDetail.vue

```diff
@@ -0,0 +1,80 @@
+<template>
```

Comment:
Ik geraak precies nooit in het detail, kan dat?

Reply:
Ja detail geraak ik ni rond in de storybook, heeft een echte bearer token nodig... al wel getest in proces ondertussen dus daar werkt het